### ISSUE TITLE
CMS_TTB_DIFF_13TEV_2016_2L_TTMNORM C-factor error

### ISSUE THREAD

**Issue by J-M-Moore** [NNPDF/nnpdf#1576]
Currently, it is not possible to run a fit using CMS_TTB_DIFF_13TEV_2016_2L_TTMNORM with the QCD C-factor. The length of the dataset is 6, whilst the length of the C-factor is 7. 

```
In [6]: len(cfdata.central_value)
Out[6]: 7

In [7]: ndata
Out[7]: 6
```

I guess this is possibly because this set is a normalised distribution, so a cut needs to be applied so that all points remain independent. Perhaps the cut has been directly included when the FK-tables were generated, rather than included in `filters.yaml`?

If this is the case, a quick fix is to remove one value from the C-factor files. But the question is... which one?

**Comment by enocera** [comment]
Hi @J-M-Moore sorry I had completely forgotten about this. I think that the last point in the C-factor should be cut away, BUT let me check this.

**Comment by J-M-Moore** [comment]
Thanks Emanuele! Yes, this is the theory I am using.

**Comment by enocera** [comment]
Closed via #1602
